lib/renderer.js

```diff
@@ -682,8 +682,14 @@
         
         //explicitly checked for visible === false. If the user can toggle the visibilty than
         //the value is always true or false. If they can't then the visible value is null or 
-        //undefined
-        if (layer.visible === false || layer.clipped || layer.type === "adjustmentLayer") {
+        //undefined. Added test for checking the stroke style, if both fill and stroke is set
+        //to false then essentially layer has no content.
+        if (layer.visible === false ||
+            layer.clipped ||
+            layer.type === "adjustmentLayer" ||
+            (layer.strokeStyle &&
+            layer.strokeStyle.fillEnabled === false &&
+            layer.strokeStyle.strokeEnabled === false)) {
```

Comment:
Yes, it's going to be part of new pull also this only fixes part of the bug. Looking into another use case which is part of same bug.